src/server/lib/app/lobby.ex

```diff
@@ -0,0 +1,102 @@
+defmodule App.Lobby do
```

Comment:
`App.Lobby` should probably be a worker of a `App.Supervisor.Lobbies`, that way we have our fault tolerance (similar to how i did `App.User` and `App.Supervisor.Users`)